plugins/scenario/World.ts

```diff
@@ -7,6 +7,8 @@ export type ForkSpec = {
   blockNumber?: number;
   allocation?: number;
   chainId?: number;
+  baseTokenFauceteer?: string;
```

Comment:
Is this where you set the `USDC whale` address?

Reply:
yes

Reply:
not sure about this

Reply:
why? it's a way to set whale account for further impersonating. We won't have faucet contracts on the mainnet.

Reply:
So token sourcer isn't finding accounts with USDC?